.github/workflows/build.yml

```diff
@@ -53,4 +53,5 @@ jobs:
       - run: make twoliter check-fmt
       - run: make twoliter check-clippy
       - run: make twoliter check-shell
+      - run: ./tools/validate-changelog.sh
```

Comment:
this action will ignore changes made solely to markdown files (L13), so if someone were to make a change that only edits the CHANGELOG, this check wouldn't run

Reply:
oh, good catch, thanks! I'll add a new workflow for validating markdown changes then